Stormshield/stormshield-endpoint-security/_meta/fields.yml

```
@@ -0,0 +1,24 @@
+stormshield.additionaldata.incidentguid:
+  type: keyword
+  description: stormshield incident guid
+  name: stormshield.additionaldata.incidentguid
+
+stormshield.specificData.targetProcess.processcommandline:
+  type: keyword
+  description: stormshield process command line
+  name: stormshield.specificData.targetProcess.processcommandline
+
+stormshield.process.user.domain:
+  type: keyword
+  description: Name of the directory the user is a member of
+  name: stormshield.process.user.domain
```

suggest using `stormshield.ses` as prefix:
 ```suggestion
stormshield.ses.process.user.domain:
  type: keyword
  description: Name of the directory the user is a member of
  name: stormshield.ses.process.user.domain
```